<launch>

    <!-- Launch the agent node -->
    <node name="agent" pkg="eurobin_coopetition" type="agent.py" output="screen"/>


</launch>